packages/inference/src/types.ts

```diff
@@ -40,7 +40,7 @@ export interface BaseArgs {
 }
 
 export type RequestArgs = BaseArgs &
-	({ data: Blob | ArrayBuffer } | { inputs: unknown }) & {
+	({ data: Blob | ArrayBuffer } | { inputs: unknown } | { image: string }) & {
```

Comment:
Arg. See https://huggingface.slack.com/archives/C016D661PAN/p1683012959767189 :(

Reply:
Apparently we should still accept `inputs: string` cc @Narsil, so let's keep using `inputs` for consistency